fe/fe-core/src/main/java/org/apache/doris/datasource/ExternalDatabase.java

```diff
@@ -283,6 +283,16 @@ public DatabaseProperty getDbProperties() {
         return dbProperties;
     }
 
+    @Override
+    public String getDbComment() {
+        throw new NotImplementedException("getDbComment() is not implemented");
```

Comment:
return empty string is better?

Reply:
Yes, you are right.

Reply:
I have modified it, please help CR again.